makefiles/targets/Linux/iphone.mk

```diff
@@ -7,20 +7,28 @@ _THEOS_TARGET_PLATFORM_SDK_NAME := iPhoneOS
 _THEOS_TARGET_PLATFORM_FLAG_NAME := iphoneos
 _THEOS_TARGET_PLATFORM_SWIFT_NAME := apple-ios
 
+SWIFTBINPATH ?= $(THEOS)/toolchain/swift/bin
+SDKBINPATH ?= $(THEOS)/toolchain/$(THEOS_PLATFORM_NAME)/$(THEOS_TARGET_NAME)/bin
+
 # Determine toolchain to use based on file existence.
-ifeq ($(SDKTARGET),)
-ifeq ($(wildcard $(THEOS)/toolchain/$(THEOS_PLATFORM_NAME)/$(THEOS_TARGET_NAME)/bin/arm64-apple-darwin14-ld),)
-SDKTARGET ?= armv7-apple-darwin11
+ifeq ($(SDKTARGETPREFIX),)
+ifeq ($(call __exists, $(SDKBINPATH)/armv7-apple-darwin11-ld),$(_THEOS_TRUE))
+SDKTARGETPREFIX ?= armv7-apple-darwin11-
+else ifeq ($(call __exists, $(SDKBINPATH)/arm64-apple-darwin14-ld),$(_THEOS_TRUE))
+SDKTARGETPREFIX ?= arm64-apple-darwin14-
 else
-SDKTARGET ?= arm64-apple-darwin14
+SDKTARGETPREFIX ?=
```

Comment:
I just realized this assignment is pretty pointless and the check on L14 is going to always be true when it doesn't have a prefix so its not adding much.

Reply:
Removed this pointless assignment.

I left the L14 check in to keep same behavior for current toolchains. Not sure how that performance gain can be added for the toolchains with no prefix.